life_expectancy/cleaning.py

```diff
@@ -1,133 +1,110 @@
 """Module to clean assignment data"""
+
 import os
 import re
 import argparse
 import pandas as pd
 from pandas import DataFrame
 
 
-def load_raw_data(
-    raw_data_file_path: str
-):
+def load_data():
```

Comment:
Leaving path to the file as an input arg. makes it better to maintain. I would recommend it as a minimum input for this function